packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -149,18 +179,17 @@ class FrameFlameChart extends CoreElement {
         ..fontWeight = 'bold'
         ..left = '0'
         ..top = '0';
-      section.add(sectionTitle);
+      cpuSection.add(sectionTitle);
 
-      drawRecursively(frame.cpuEventFlow, 0, section);
+      drawRecursively(frame.cpuEventFlow, 0, cpuSection);
     }
 
     void drawGpuEvents() {
-      final section = div(c: 'flame-chart-section');
-      add(section);
+      gpuSection = div(c: 'flame-chart-section');
```

Comment:
it seems like we are recreating a bunch of elements on re-render. It would be best to just change the CSS styles to update all the element positions and leave the element tree alone.

Reply:
Changed to manually updating the left/width attributes but not re-drawing the elements.